src/main/resources/static/js/signup-account.js

```diff
@@ -0,0 +1,26 @@
+document.getElementById("signup-form").addEventListener("submit", (event) => {
+    event.preventDefault()
+    const validInputValues = checkSignupValidity()
+    if (validInputValues) {
+        /// Create the new user in database ?? Either json or form data ?? 
+    } else {
+        /// Show message that input values are not correct
+    }
+})
+function checkSignupValidity() {
```

Comment:
Rewrite the method to something analogical to the checkNewPasswordInputsValidity() function in the settings-account.js file

Reply:
Sorry did you mean analgous to the other function. So the same sort of behaviour ? e.g. setCustomValidity("Passwords dont match" if the dont match and setCustomValidity("") if they are the same.

Reply:
Yes, that's exactly the behaviour I meant. And add this function to 'input' event listeners of both password fields.